### ISSUE TITLE
Tengine Support Operators List : Float32, Float16, UInt8, Int8

### ISSUE THREAD

**Issue by BUG1989** [OAID/Tengine#464]
## Tengine Lite 算子实现统计

| OP Name              | Float32 | Float16 | UInt8 | Int8 |
| -------------------- | ------- | ------- | ----- | ---- |
| Absval               | √       |         |       |      |
| Add_n                | √       |         |       |      |
| ArgMax               | √       |         | √     |      |
| ArgMin               | √       |         | √     |      |
| BatchNormalize       | √       |         | √     |      |
| Batchtospacend       | √       |         |       |      |
| Bias                 | √       |         |       |      |
| BroadMul             | √       |         |       |      |
| Cast                 | √       |         |       |      |
| Ceil                 | √       |         | √     |      |
| Clip                 | √       |         | √     |      |
| Comparison           | √       |         |       |      |
| Concat               | √       | √       | √     | √    |
| Convolution          | √       | √       | √     | √    |
| Const                | √       | √       | √     | √    |
| Crop                 | √       |         | √     |      |
| Deconvolution        | √       |         |       |      |
| Depthtospace         | √       |         |       |      |
| DetectionOutput      | √       |         | √     | √    |
| DetectionPostProcess | √       |         | √     |      |
| Dropout              | √       | √       | √     | √    |
| Eltwise              | √       |         | √     | √    |
| Elu                  | √       |         | √     |      |
| Embedding            | √       |         |       |      |
| Expanddims           | √       |         |       |      |
| FullyConnected       | √       | √       | √     | √    |
| Flatten              | √       |         | √     | √    |
| Gather               | √       |         | √     |      |
| Gemm                 |         |         |       |      |
| Gru                  | √       |         |       |      |
| HardSigmoid          | √       |         |       |      |
| Hardswish            | √       |         |       |      |
| InputOp              | √       | √       | √     | √    |
| InstanceNorm         | √       |         |       |      |
| Interp               | √       |         |       |      |
| Logical              | √       |         |       |      |
| Logistic             | √       |         |       |      |
| Lrn                  | √       |         |       |      |
| Lstm                 | √       |         |       |      |
| Matmul               | √       |         |       |      |
| Maximum              | √       |         |       |      |
| Mean                 | √       |         |       |      |
| Minimum              | √       |         |       |      |
| Mvn                  | √       |         |       |      |
| Noop                 | √       |         |       |      |
| Normalize            | √       |         |       |      |
| Pad                  | √       |         | √     |      |
| Permute              | √       |         | √     |      |
| Pooling              | √       | √       | √     | √    |
| PReLU                | √       |         | √     | √    |
| PriorBox             | √       |         | √     | √    |
| Psroipooling         | √       |         |       |      |
| ReduceL2             | √       |         |       |      |
| Reduction            | √       |         |       |      |
| Region               | √       |         |       |      |
| ReLU                 | √       | √       | √     | √    |
| ReLU6                | √       |         | √     | √    |
| Reorg                | √       |         |       |      |
| Reshape              | √       |         | √     | √    |
| Resize               | √       |         |       |      |
| Reverse              | √       |         |       |      |
| RNN                  | √       |         |       |      |
| Roialign             | √       |         |       |      |
| RoiPooling           | √       |         |       |      |
| Round                | √       |         |       |      |
| Rpn                  | √       |         |       |      |
| Scale                | √       |         |       |      |
| Selu                 | √       |         | √     |      |
| ShuffleChannel       | √       |         | √     | √    |
| Sigmoid              | √       |         | √     |      |
| Slice                | √       |         |       |      |
| Softmax              | √       |         |       | √    |
| Spacetobatchnd       | √       |         | √     |      |
| Spacetodepth         | √       |         | √     |      |
| SparseToDense        | √       |         |       |      |
| Split                | √       |         |       |      |
| SquaredDifference    | √       |         | √     |      |
| Squeeze              | √       |         | √     |      |
| StridedSlice         | √       |         | √     |      |
| SwapAxis             | √       |         | √     |      |
| Tanh                 | √       |         | √     |      |
| Threshold            | √       |         | √     |      |
| Topkv2               | √       |         | √     |      |
| Transpose            | √       |         | √     |      |
| Unary                | √       |         | √     |      |
| Unsqueeze            | √       |         | √     |      |
| Upsample             | √       |         | √     |      |
| ZerosLike            | √       |         | √     |      |
| Mish                 | √       |         |       |      |
